lib/initial_page.dart

```diff
@@ -34,6 +36,17 @@ class _InitialPageState extends State<InitialPage> {
     }
   }
 
+  void _clearSharedPreferences() async {
+    final sharedPrefs = await SharedPreferences.getInstance();
+    await sharedPrefs.remove('destinationId');
+    await sharedPrefs.remove('projectId');
+  }
```

Comment:
this method is duplicated in another place, would be better if you could move it to a helper/utility class?